plugins/serializers/registry.go

```diff
@@ -73,6 +77,8 @@ func NewSerializer(config *Config) (Serializer, error) {
 		serializer, err = NewGraphiteSerializer(config.Prefix, config.Template, config.GraphiteTagSupport)
 	case "json":
 		serializer, err = NewJsonSerializer(config.TimestampUnits)
+	case "splunkmetric":
```

Comment:
If we renamed this "splunk" would it cause confusion about the format type?  Since all of the formats are for encoding metrics I feel like the metric suffix may be redundant.

Reply:
I named it splunkmetric because Splunk has two (2) different index types, events and metrics. This is more of an indication that telegraf is feeding a Splunk metrics index vs. an events index. I'd be OK changing it if you really want to.

Reply:
Thanks for the explanation, I think your decision to call it `splunkmetric` is the right one then.

Reply:
👍